_config.yml

```diff
@@ -17,7 +17,7 @@ title: ChickTech Favorite Foods
 email: bihan.zh@gmail.com
 description: >- # this means to ignore newlines until "baseurl:"
   What are some of ChickTechRDU's favorite foods?
-baseurl: "" # the subpath of your site, e.g. /blog
+baseurl: ""
```

Comment:
See if you can put this line back the way it was, including the original comment following the ``#``